lib/rubocop/cop/minitest/no_test_cases.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks if test class contains any test cases.
+      #
+      # @example
+      #   # bad
+      #   class FooTest < Minitest::Test
+      #     def do_something
+      #     end
+      #   end
+      #
+      #   # good
+      #   class FooTest < Minitest::Test
+      #     def test_something
+      #       assert true
+      #     end
+      #   end
+      #
+      class NoTestCases < Base
+        include MinitestExplorationHelpers
+
+        MSG = 'Test class has no test cases.'
```

Comment:
IMO, I think the action that user should take becomes more explicit:

```suggestion
        MSG = 'Test class should have test cases.'
```